coalib/bearlib/aspects/collections.py

```diff
@@ -7,7 +8,23 @@ class aspectlist(list):
     """
 
     def __init__(self, seq=()):
-        super().__init__(map(assert_aspect, seq))
+        """
+        Initialize a new aspectlist.
+
+        >>> from .root import Root
+        >>> from .Spelling import Spelling
+        >>> aspectlist([Root.Spelling.coalaCorrect, Root.Spelling.aspectsYEAH])
```

Comment:
https://github.com/coala/coala/issues/4401 should be fixed first, otherwise lots of unnecessary changes occur.